.config/nvim/core/init.lua

```diff
@@ -0,0 +1 @@
+require("core.keybindings")
```

Comment:
I think we need to import `core.options` too here, WDYT?

Reply:
You're right. I've added it in f4ca17b.